setup.md

```diff
@@ -8,72 +8,156 @@ permalink: /setup/
 This guide describes how to set up your development environment to run Flutter apps on
 iOS or Android.
 
-* TOC Placeholder
-{:toc}
+New to GitHub? First [install the software](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git){: alert-link}
```

Comment:
+1 on removing this. It's fairly common to omit it on most open source software instructions that uses git.